src/assets/gateway.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
What is this a picture of?

Reply:
I'm not sure what it is, but its from the design. Looks like a person wearing 70's glasses standing in front of a bullseye.

Reply:
it was a stock photo that got left in the design, but I've removed all these logos anyways.